src/coreclr/vm/common.h

```diff
@@ -397,7 +397,4 @@ extern DummyGlobalContract ___contract;
 #endif
 
 void LogErrorToHost(const char* format, ...);
-
```

Comment:
A nit - can we revert the change in this file as it is a pure whitespace change?